requirements.txt

```diff
@@ -1,4 +1,5 @@
 aiohttp==2.3.9
 google-auth==1.3.0
+google-cloud-pubsub==0.30.1
```

Comment:
This is not yet used in the `GEventMessage` below, but I use it for tests. It does get used in the next commit.